lib/ProductOpener/Display.pm

```diff
@@ -7143,11 +7143,19 @@ HTML
 		$other_editors = "<br>\n$Lang{also_edited_by}{$lang} ${other_editors}.";
 	}
 
+	my $checked = "";
+	if ((defined $product_ref->{checked}) and ($product_ref->{checked} eq 'on')) {
+		my $last_checked_date = display_date_tag($product_ref->{last_checked_t});
+		my $last_checker = "<a href=\"" . canonicalize_tag_link("users", get_fileid($product_ref->{last_checker})) . "\">" . $product_ref->{last_checker} . "</a>";
+		$checked = "<br/>\n$Lang{product_last_checked}{$lang} $last_checked_date $Lang{by}{$lang} $last_checker.";
```

Comment:
Should we maybe change this translation to use sprintf? The sentence structure may be different in other languages than the current "Last check of product page on" folowed by the three values. @teolemon

Reply:
For this one I just copied what we had for the last_edit_date. It's a good point. I'm not sure if sprintf is the good solution, the %s might be confusing for translators. Maybe we could use <tags> instead. e.g "Last verification by <user> on <date>", that way the word order could be changed if needed.

This is something we should probably discuss separately and apply across all the strings. cc @teolemon